### ISSUE TITLE
Implement HTMLLabelElement.prototype.control & unit test

### ISSUE THREAD

**Issue by termi** [Raynos/DOM-shim#34]
http://www.w3.org/TR/html5/forms.html#dom-label-control
https://developer.mozilla.org/en/DOM/HTMLLabelElement

**We need unit test for this**

**Comment by Raynos** [comment]
yes/no. Currrent DOM-shim roadmap is.
- rewrite the bloody tests
- make DOM4 compliance work on IE6+ 
- Implement DOM3 events
- Implement XHR2
- Implement everything from the HTML WHATWG spec

This means any editions to HTMLXElement are low on the priority lists. Patches are welcome ;)

**Comment by termi** [comment]
Good. I am just working on it

**Comment by termi** [comment]
So. May I close this is issue?

**Comment by Raynos** [comment]
Yes, it's been fixed, I believe. Although we don't have unit tests for this do we?

**Comment by termi** [comment]
I can write it. Need to check it out if it already been written.

I this case I need to open separate issue for 'Add HTMLLabelElement.prototype.control unit tests', or I can edit this issue detail?

**Comment by Raynos** [comment]
Just put the details here. 

I added you as a collaborator so you should be able to pull/push to the repository directly. Which means you won't have to open pull requests for everything.

Although do open pull requests for big things.

**Comment by termi** [comment]
Cool!
I will try use this privilege carefully
